docs/source/containerization/kubernetes.mdx

```diff
@@ -64,10 +64,10 @@ kind: ServiceAccount
 metadata:
   name: release-name-router
   labels:
-    helm.sh/chart: router-1.0.0-rc.5
+    helm.sh/chart: router-1.0.0-rc.7
```

Comment:
These seem to be "extra" spaced out because the `kubernetes.mdx` was not updated during the last release.